From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ASTAR <apo2073@outlook.com>
Date: Sat, 24 Jan 2026 22:26:30 +0900
Subject: [PATCH] add Fabric


diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index 7a5bda878ff77d20a9ea11e8ac879cebcec2f63d..459c5de049fda9c7fedb4568f0ae58c4d20e6229 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -516,6 +516,8 @@ public final class CraftServer implements Server {
         datapackManager = new io.papermc.paper.datapack.PaperDatapackManager(console.getPackRepository()); // Paper
         this.spark = new io.papermc.paper.SparksFly(this); // Paper - spark
         org.leavesmc.leaves.protocol.core.LeavesProtocolManager.init(); // Leaves - protocol
+        io.github.grassproject.petal.bridge.PetalBridge.attachMinecraftServer(console); // Petal
+        io.github.grassproject.petal.bridge.PetalBridge.attachCraftServer(this); // Petal
     }
 
     public boolean getCommandBlockOverride(String command) {
